src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private ArrayList<String> readFile(String fileName) {
+        File fileToRead = new File(fileName);
+
+        try (BufferedReader br = new BufferedReader(new FileReader(fileToRead))) {
+            ArrayList<String> operationsList = new ArrayList<>();
+            String line;
+
+            while ((line = br.readLine()) != null) {
+                operationsList.add(line);
+            }
+
+            return operationsList;
+        } catch (IOException e) {
+            throw new RuntimeException("File not found: ", e);
```

Comment:
The exception message 'File not found: ' is not entirely accurate since IOException covers more than just a missing file. Consider a more generic message like 'Can't read from the file: '.